Chore: (intro to storybook) restructure create addons section (#344)
chromaui/learnstorybook.com: content/intro-to-storybook/react/en/creating-addons.md

commit: 'bebba5d'
---

> Throughout this tutorial we were introduced to one of the key features of Storybook, its robust system of [addons](https://storybook.js.org/addons/introduction/), which can be used to enhance not only yours but also your team's developer experience and workflows.

Reviewer: ```suggestion
Earlier, we introduced a key Storybook feature: the robust [addons](https://storybook.js.org/addons/introduction/) ecosystem. Addons are used to enhance your developer experience and workflows.
```

Author: Changed with the latest commit. Marking it as solved
